pkgs/tools/system/lshw/default.nix

```diff
@@ -1,27 +1,33 @@
-{ stdenv, fetchurl
-, withGUI? false, gtk? null, pkgconfig? null, sqlite? null  # compile GUI
- }:
-stdenv.mkDerivation rec {
+{ stdenv, lib, fetchurl, fetchpatch
+, withGUI ? false, gtk2, pkgconfig, sqlite # compile GUI
```

Comment:
I don't fully understand the benefits of writing `nulls` as `callPackage` passes all arguments anyway (as @joachifm has mentioned) and Nix is a non-strict language.
Maybe there are cases when it's handy?